socketshark/__init__.py

```diff
@@ -126,35 +127,32 @@ async def prepare(self):
 
         Initialize Redis connection and the receiver class.
         """
-        redis_receiver = Receiver(loop=asyncio.get_event_loop())
-        redis_settings = self.config['REDIS']
+        redis_settings = [self.config['REDIS']]
+        if self.config.get('REDIS_ALT'):
+            redis_settings.append(self.config['REDIS_ALT'])
+        self.redis_connections = []
```

Comment:
Should we add this to `SocketShark.__init__` to make this more discoverable? Seems too hidden to add an important attribute to an instance within one of the methods.

Reply:
This is done in [318608f](https://github.com/closeio/socketshark/pull/105/commits/318608faf0f80511ec41098119f746b2a0257090)